Updated What's New (#3045)
grommet/hpe-design-system: aries-site/src/pages/whats-new.mdx

<!-- BEGIN: What's New Entries -->

## November 2022

- **Avatar**

>   - We've added Avatar [guidance and examples](/components/avatar) and [Figma component](https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1&t=bQK90xgXcu9vlYYm-0).

Reviewer: new link from Kenny

<!-- BEGIN: What's New Entries -->

## November 2022

- **Avatar**

>   - We've added Avatar [guidance and examples](https://design-system.hpe.design/components/avatar) and [Figma component](https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1&t=bQK90xgXcu9vlYYm-0).

Reviewer: There's nothing in the figma file for the finished? @KennyAtHPE

Reviewer 2: Try this [link](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=3027%3A61406&t=CpM6ERbKsy0lCo2p-1) instead

>   - We've added Avatar [guidance and examples](https://design-system.hpe.design/components/avatar) and [Figma component](https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1&t=bQK90xgXcu9vlYYm-0).

Reviewer 3: For smoother page transitions ---- For internal URLs, NextJS will trigger a true page reload when provided a complete URL; it will do a render without page reload when provided just the path.

```suggestion
  - We've added Avatar [guidance and examples](/components/avatar) and [Figma component](https://www.figma.com/file/d7a74LgBABdgTZR6HrzWKo/HPE-Avatar?node-id=0%3A1&t=bQK90xgXcu9vlYYm-0).
```

- **Wizard**

>   - We've added Wizard [Anatomy](https://design-system.hpe.design/templates/wizard#anatomy) guidance.

Reviewer 3: ```suggestion
  - We've added Wizard [anatomy](/templates/wizard#anatomy) guidance.
```

## October 2022

- **HPE Brand Refresh**

>   - We've published a new [grommet-theme-hpe](https://www.npmjs.com/package/grommet-theme-hpe) with the latest HPE brand styling.

Author: I don't think we have release notes for the theme, do we?

>   - We've published a new [grommet-theme-hpe](https://www.npmjs.com/package/grommet-theme-hpe) with the latest HPE brand styling.

Reviewer: https://github.com/grommet/grommet-theme-hpe/releases/tag/v4.0.0 
yes we make those notes as well

>   - We've published a new [grommet-theme-hpe](https://www.npmjs.com/package/grommet-theme-hpe) with the latest HPE brand styling.

Author: Done

>   - We've added [Distinctive Brand Assets](https://design-system.hpe.design/foundation/distinctive-brand-assets) guidance and examples.

Reviewer 3: ```suggestion
  - We've added [distinctive brand assets](/foundation/distinctive-brand-assets) guidance and examples.
```

  - We've added a extra Button kinds for 'cta-primary' and 'cta-alternate' along with [guidance](https://design-system.hpe.design/components/button#cta-button) for using them.
  - We've clarified how to use the [Call-to-action Button](https://design-system.hpe.design/components/button#when-to-use-a-cta-button).

>   - We've updated the site to follow the HPE capitalization guidance for using mostly sentence case.

Reviewer: should we link the guide?

Author: Good idea! I couldn't find anything for this though. I searched the site for "capitalization" and "sentence case" but didn't get any matches for them.

Reviewer: https://design-system.hpe.design/foundation/voice-and-tone#capitalization

i wonder if we need to update our tags to find it

Author: Done

- **Header**

>   - We've added guidance on various types of [Headers](https://design-system.hpe.design/components/header#when-to-use).

Reviewer 3: ```suggestion
  - We've added guidance on various types of [Headers](/components/header#when-to-use).
```

- **CheckBox**

>   - We've enhanced guidance on using [CheckBox outside of FormField](https://design-system.hpe.design/components/checkbox#checkbox-outside-of-formfield).

Reviewer 3: ```suggestion
  - We've enhanced guidance for using a [CheckBox outside of a FormField](/components/checkbox#checkbox-outside-of-formfield).
```

- **Menu**
  - We've updated the styling for [grouped Menu items](https://www.figma.com/file/DyFWlzLBUtK5KH5jJhwoCB/HPE-Menu-Component?node-id=606%3A1489&t=5s9S2IY0lW34lv9J-0).
- **Feedback**
  - We've added an enhanced Feedback mechanism on the HPE Design System site. Hint, there's a button in the bottom right corner on every page. Check it out! This is integrated with Qualtrics.

## September 2022

- **Card**

>   - We've enhanced [Card](https://design-system.hpe.design/components/card) guidance and examples.

Reviewer 3: ```suggestion
  - We've enhanced [Card](/components/card) guidance and examples.
```

  - We've added [Call to action card](https://design-system.hpe.design/components/card/call-to-action-card) and [Navigational xard](https://design-system.hpe.design/components/card/navigational-card) examples and guidance.
- **SelectMultiple**
  - We've added a [SelectMultiple component](https://design-system.hpe.design/components/selectmultiple).

## August 2022

- **Form**

>   - We've added guidance on [Where to present a form](https://deploy-preview-2834--keen-mayer-a86c8b.netlify.app/templates/forms#where-to-present-a-form)

Reviewer: ```suggestion
  - We've added guidance on [Where to present a form](https://design-system.hpe.design/templates/forms#where-to-present-a-form)
```

>   - We've added guidance on [Where to present a form](https://design-system.hpe.design/templates/forms#where-to-present-a-form)

Reviewer 3: ```suggestion
  - We've added guidance on [where to present a form](/templates/forms#where-to-present-a-form).
```

>   - We've added examples and guidance for [Full-page forms](https://deploy-preview-2834--keen-mayer-a86c8b.netlify.app/templates/forms#full-page-forms)

Reviewer: ```suggestion
  - We've added examples and guidance for [Full-page forms](https://design-system.hpe.design/templates/forms#full-page-forms)
```

>   - We've added examples and guidance for [Full-page forms](https://design-system.hpe.design/templates/forms#full-page-forms)

Reviewer 3: ```suggestion
  - We've added examples and guidance for [full-page forms](/templates/forms#full-page-forms).
```

- **DataTable**

>   - We've created a [Figma component](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2915%3A56106&t=ZMldJMLzCjzI8VxQ-0) for DataTable cusomtization.

Reviewer 3: ```suggestion
  - We've created a [Figma component](https://www.figma.com/file/5uOrsL2qk8XwwH8C1ZpDb6/HPE-Design-System-Components?node-id=2915%3A56106&t=ZMldJMLzCjzI8VxQ-0) for DataTable customization.
```

- **Analytics**

>   - We've made it easier to collect analytics data via `onAnalytics` and the Grommet component.

Author: We haven't published those yet. We were waiting for more real-world usage to verify the API first.

>   - We've made it easier to collect analytics data via `onAnalytics` and the Grommet component.

Reviewer: aw okay no worries then

>   - We've made it easier to collect analytics data via `onAnalytics` and the Grommet component.

Reviewer 3: ```suggestion
  - We've made it easier to collect web analytics data (e.g navigation, layer and form interactions, and more) via `onAnalytics` and the Grommet component.
```

## July 2022

- **Form**

>   - We've added guidance on accepting [Terms and Conditions](https://design-system.hpe.design/templates/forms#terms-and-conditions)

Reviewer 3: ```suggestion
  - We've added guidance on accepting [Terms and Conditions](/templates/forms#terms-and-conditions).
```
